hint.cabal

```diff
@@ -40,14 +40,13 @@ test-suite unit-tests
                   HUnit,
                   directory,
                   filepath,
-                  extensible-exceptions,
                   exceptions >= 0.10.0,
                   stm,
 
                   -- packages used by setImports calls
                   containers
 
-  if impl(ghc >= 8.4 && < 8.9) {
+  if impl(ghc >= 8.4) {
```

Comment:
Come to think of it, since hint only supports ghc-8.6 and up, this condition is always satisfied, so I should just remove it and always use the code in the `#ifdef DNEED_PHANTOM_DIRECTORY` blocks!